Default (Linux).sublime-keymap

```diff
@@ -1,5 +1,6 @@
 [
     { "keys": ["tab"], "command": "hippie_word_completion", "context": [
+        { "key": "read_only", "operator": "not_equal" },

```

Comment:
```suggestion
        { "key": "read_only", "operand": false },
```
More obvious, and same as all the other. Same below.

Reply:
Doesn't work.  I thought I'm crazy.  ST4 bug? Or I *am* crazy.

Reply:
Maybe the key is only defined if a view is read only, inconvenient. Looks like a bug in ST4.